### ISSUE TITLE
Bug  Thalorien Dawnseeker

### ISSUE THREAD

**Issue by giovannilol88** [dalaranwow/dalaran-wow#5903]
[//]: # (INFORMATION ABOUT TAGS) Thalorien Dawnseeker
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. Thalorien Dawnseeker quest. I cant complete bug quest
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Golganeth** [comment]
Invalid report, no valid info + this quest works just fine, several players have completed it during the last days.
Closing up

**Comment by giovannilol88** [comment]
you kidding me?

Il giorno sab 27 giu 2020 alle ore 14:23 Golganeth <notifications@github.com>
ha scritto:

**Comment by giovannilol88** [comment]
![image](https://user-images.githubusercontent.com/67504419/85922387-19b8f600-b883-11ea-8f53-01854a6cf13e.png)

**Comment by giovannilol88** [comment]
![image](https://user-images.githubusercontent.com/67504419/85922419-5553c000-b883-11ea-8ef7-bcd248317f77.png)
